src/action/macos/create_volume_service.rs

```diff
@@ -51,39 +57,41 @@ impl CreateVolumeService {
 
         // If the service is currently loaded or running, we need to unload it during execute (since we will then recreate it and reload it)
         // This `launchctl` command may fail if the service isn't loaded
-        let mut check_loaded_command = Command::new("launchctl");
-        check_loaded_command.arg("print");
-        check_loaded_command.arg(format!("system/{}", this.mount_service_label));
-        tracing::trace!(
-            command = format!("{:?}", check_loaded_command.as_std()),
-            "Executing"
-        );
-        let check_loaded_output = check_loaded_command
-            .output()
-            .await
-            .map_err(|e| ActionErrorKind::command(&check_loaded_command, e))
-            .map_err(Self::error)?;
-        this.needs_bootout = check_loaded_output.status.success();
-        if this.needs_bootout {
+        let check_loaded_output = execute_command(
```

Comment:
Should we, like, I dunno, abstract this into a reusable function? (also, output / NNN)